opennlp-tools/src/main/java/opennlp/tools/formats/conllu/ConlluSentenceSampleStreamFactory.java

```diff
@@ -28,6 +28,9 @@
 import opennlp.tools.util.InputStreamFactory;
 import opennlp.tools.util.ObjectStream;
 
+/**
+ * <b>Note:</b> Do not use this class, internal use only!
+ */
```

Comment:
In Apache Commons I think it is common to have an "internal" subpackage for classes that must not be used by external users. Maybe we should move these classes too? That way you could quickly identify use of internal API classes in other projects.

Reply:
Could be done via a separate JIRA issue. Feel free to open it and propose that change. Could be realized quickly, I guess.

Reply:
These classes are in OpenNLP for some time now and the internal use only documentation was missing (until now). If we move these classes to an `internal` package, we might break users, which (falsely) relied on these classes being "open for the public". I like the suggestion by @mawiesne to make that change happen in a separate JIRA, so we can clearly document, that our users might be impacted.